developer/developer-faqs.md

```diff
@@ -14,7 +14,17 @@ Reaction tries to be as modular as possible. All user customizations are intende
 
 Ecommerce and marketplace sites from around the world are using Reaction in production now. Check out our [Community Showcase](https://reactioncommerce.com/community-showcase).
 
+### Can I compile a mobile app from Reaction? Are you going to develop a mobile app?
 
+While theoretically you can compile a [Meteor app to mobile using Cordova](https://guide.meteor.com/mobile.html) we don't support
```

Comment:
While Meteor allows all apps to be compiled into Android and iOS apps with Cordova, we do not support it. Setting up a Reaction app with Cordova may be difficult. Learn more about building [mobile apps with Meteor's Cordova integration](https://guide.meteor.com/mobile.html).

While we do have plans to work on a mobile app on the [Roadmap](https://reactioncommerce.com/roadmap), we expect to use a non-Cordova approach, like React Native. We do not have a release date.

What's important to know, however, is that Reaction Commerce takes a "mobile first" design approach to ensure that both the consumer-facing and admin-facing sites are functional on most modern mobile devices.